docs/Sources.md

```diff
@@ -39,6 +39,7 @@ source:
   default: sql
   sql:
     adapter: mysql2
+    host: server_host
```

Comment:
Specifying a host that isn't localhost is indeed a common use case, so this enhances the example!